src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -140,6 +140,10 @@ Apply to alert to override Bootstrap's default 3d alerts
     padding-right: 15px;
 }
 
+div[class*="col-"][class*="col-no-padding"] {
+    padding: 0;
+}
```

Comment:
I added a new generic CSS rule which can be used anywhere when we want to remove the padding that comes along with a bootstrap column